sdk/sdk-api/src/main/java/mil/emp3/api/MirrorCache.java

```diff
@@ -34,23 +34,42 @@
 import mil.emp3.mirrorcache.spi.MirrorCacheClientProviderFactory;
 
```

Comment:
This would be a great place to explain how MirrorCache works and what Clients applications need to do to activate it. Any limitations and known issues can also be documented here. Is there a README.md in mirror-cache repository that describes this?